src/3d/DiffusionType.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef DIFFUSION_TYPE_H
+#define DIFFUSION_TYPE_H
+
+
+/**
+ *
+ * The DiffusionType enum is used to distinguish between different anisotropic diffusion behaviors:
+ * - ISOTROPIC: Uniform diffusion in all directions.
+ * - CIRCUMFERENTIAL: Diffusion primarily in the circumferential direction.
+ * - RADIAL: Diffusion primarily in the radial direction.
+ * 
+ * Note: We did not implement the axon-based anisotropic diffusion model, due to lack of real-world data.
+ */
+
+
+enum DiffusionType {
```

Comment:
You should consider using `enum class`es in place of `enum`s, since they are more type-safe.